cpplint.py

```diff
@@ -4736,6 +4850,7 @@ def _ClassifyInclude(fileinfo, include, is_system):
   # This is a list of all standard c++ header files, except
   # those already checked for above.
   is_cpp_h = include in _CPP_HEADERS
+  is_c_h = include in _C_HEADERS or Search(r'sys\/.*\.h', include)
```

Comment:
A code omment on `Search(r'sys\/.*\.h', include)` would be nice.

Reply:
Done